### ISSUE TITLE
Add support for decimal string format to generator

### ISSUE THREAD

**Issue by Tetraquark** [icerockdev/moko-network#96]
Now if openapi config has property like this:

```
amount:
  title: Amount
  type: string
  format: decimal
```

Then the class with this property will be generated with error:

```kotlin
@SerialName("amount")
val amount: java.math.BigDecimal // can't import java in common code
```

**Comment by Alex009** [comment]
will generate BigNum type from https://korlibs.soywiz.com/kbignum/

**Comment by Alex009** [comment]
will be available in 0.12.0
